### PR TITLE
Graph ql

### PR DESCRIPTION
**Pull request by ArifPMan**

### CODE REVIEW

**Comment by ArifPMan** [comment]
Please review and provide f/b

File: v6/postman/sending_api_requests/graphql.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+### Importing GraphQL schemas 
+
+You can easily create or import GraphQL schemas in Postman. To create a GraphQL API schema in Postman, perform the steps illustrated in the following screen:
+
+[![auto complete](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/GraphQL-Auto1.gif)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/GraphQL-Auto1.gif)
+
+1. Click **APIs** tab, select an API (or create a new API), navigate to the **Define** tab, select **Create new**/**Import file**, choose *GraphQL* from the dropdown and click **Save**.  
+
+2. Go to the **Collections** tab, navigate to the **Body** tab, select **GraphQL** body type, and start typing your query and autocomplete lets you view and insert possible completions from the content in your GraphQL schema.
```
Comment (numaanashraf) [inline v6/postman/sending_api_requests/graphql.md]: this line sounds confusing. where is the "body" tab after the user navigate to the "collections" tab?

Comment (numaanashraf) [reply v6/postman/sending_api_requests/graphql.md]: "start typing your query and autocomplete lets you view"
autocomplete wont work till user selects the schema from the dropdown

**Comment by numaanashraf** [comment]
@ArifPMan I've provided my review comments

File: v6/postman/sending_api_requests/graphql.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+* [Sending a GraphQL query](#sending-a-graphql-query)
+* [Working with GraphQL variables](#working-with-graphql-variables)
+* [Importing GraphQL schemas](#importing-graphql-schemas)
+* [Autocomplete for GraphQL](#autocomplete-for-graphql)
+
+### Sending a GraphQL query
+
+Authoring and sending a GraphQL query in Postman is simple. The following screen illustrates sending a GraphQL query in Postman using the new body type *GraphQL*:
+
+[![binary data](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/GraphQL-Body.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/GraphQL-Body.png)
```
[the remark this answers is not in the document]
Comment (ArifPMan) [reply v6/postman/sending_api_requests/graphql.md]: resolved

File: v6/postman/sending_api_requests/graphql.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+To send a GraphQL query, ensure you're present in the *Body* tab and then select **GraphQL** (shown above in red circle). Enter your GraphQL query in the *Query* section and send your request. The GraphQL body type provides a rich user interface for authoring and sending requests. The Editor enables you to author both queries and variables separately, as illustrated in the above screen. 
+
+### Working with GraphQL variables
+
+Postman provides a separate interface to author GraphQL variables. You can also use Postman variables as data for GraphQL variables using `{{variable}}` syntax.
+
+After defining your GraphQL query in the *Query* editor, you can author and edit GraphQL variables in a separate variables editor, as illustrated below:
+
+[![binary data](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/GraphQL-Body-Variables.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/GraphQL-Body.png)
```
[the remark this answers is not in the document]
Comment (ArifPMan) [reply v6/postman/sending_api_requests/graphql.md]: Two are different screenshots. first one does not have variables.

File: v6/postman/sending_api_requests/graphql.md (hunk 1 above)
Comment (ArifPMan) [reply v6/postman/sending_api_requests/graphql.md]: resolved.

**Comment by ArifPMan** [comment]
@numaanashraf I've resolved all the comments.

File: v6/postman/sending_api_requests/graphql.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+* Edit and author GraphQL variables in a separate view
+
+This section describes the following topics:
+
+* [Sending a GraphQL query](#sending-a-graphql-query)
+* [Working with GraphQL variables](#working-with-graphql-variables)
+* [Importing GraphQL schemas](#importing-graphql-schemas)
+* [Autocomplete for GraphQL](#autocomplete-for-graphql)
+
+### Sending a GraphQL query
```
Comment (tristandenyer) [inline v6/postman/sending_api_requests/graphql.md]: ### is for H3. At this point in the doc, this should be an H2 (##).

File: v6/postman/sending_api_requests/graphql.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+Authoring and sending a GraphQL query in Postman is simple. The following screen illustrates sending a GraphQL query in Postman using the new body type *GraphQL*:
+
+[![graphql body](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/GraphQL-Body.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/GraphQL-Body.png)
+
+To send a GraphQL query, ensure you're present in the *Body* tab and then select **GraphQL** (shown above in red circle). Enter your GraphQL query in the *Query* section and send your request. The GraphQL body type provides a rich user interface for authoring and sending requests. The Editor enables you to author both queries and variables separately, as illustrated in the above screen. 
+
+### Working with GraphQL variables
```
Comment (tristandenyer) [inline v6/postman/sending_api_requests/graphql.md]: ### is for H3. At this point in the doc, this should be an H2 (##).

File: v6/postman/sending_api_requests/graphql.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+Postman provides a separate interface to author GraphQL variables. You can also use Postman variables as data for GraphQL variables using `{{variable}}` syntax.
+
+After defining your GraphQL query in the *Query* editor, you can author and edit GraphQL variables in a separate variables editor, as illustrated below:
+
+[![edit variables](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/GraphQL-Body-Variables.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/GraphQL-Body-Variables.png)
+
+### Importing GraphQL schemas
```
Comment (tristandenyer) [inline v6/postman/sending_api_requests/graphql.md]: ### is for H3. At this point in the doc, this should be an H2 (##).

File: v6/postman/sending_api_requests/graphql.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+You can easily create or import GraphQL schemas in Postman. To create a GraphQL API schema in Postman, perform the steps illustrated in the following screen:
+
+[![auto complete](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/GraphQL-Auto1.gif)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/GraphQL-Auto1.gif)
+
+1. Click **APIs** tab, select an API (or create a new API), navigate to the **Define** tab, select **Create new**/**Import file**, choose *GraphQL* from the dropdown and click **Save**.  
+
+2. Go to the **Collections** tab, select the request, navigate to the **Body** tab, and select **GraphQL** body type. Select your schema from the **GraphQL (Draft)** dropdown and start typing your query. Autocomplete lets you view and insert possible completions from the content in your GraphQL schema. 
+
+### Autocomplete for GraphQL
```
Comment (tristandenyer) [inline v6/postman/sending_api_requests/graphql.md]: ### is for H3. At this point in the doc, this should be an H2 (##).

Comment (ArifPMan) [reply v6/postman/sending_api_requests/graphql.md]: Resolved comments @tristandenyer

File: v6/postman/sending_api_requests/graphql.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (ArifPMan) [reply v6/postman/sending_api_requests/graphql.md]: We are using two different screenshots...

**Pull request merged**